vitrina/datasets/forms.py

```diff
@@ -362,6 +362,58 @@ def __init__(self, request=None, organization=None, *args, **kwargs):
         )
 
 
+class InformationSystemResourceForm(BaseResourceForm):
+    identifier = forms.CharField(label=_("Identifikatorius"), required=False)
+
+    class Meta:
+        model = Dataset
+        fields = (
+            "title",
+            "description",
+            "is_public",
+            "tags",
+            "catalog",
+            "frequency",
+            "access_rights",
+            "files",
+            "name",
+            "contact",
+            "creator",
+            "publisher",
+            "managed_by_publisher",
+            "landing_page",
+        )
+
+    def __init__(self, request=None, organization=None, *args, **kwargs):
+        super().__init__(request, organization, *args, **kwargs)
+        instance = self.instance if self.instance and self.instance.pk else None
+        if instance:
+            self.fields["identifier"].initial = instance.identifier if instance.identifier else ""
+
+        self.helper.layout = Layout(
+            Field("is_public", placeholder=_("Ar duomenys vieši")),
+            Field("title", placeholder=_("Duomenų rinkinio pavadinimas")),
+            Field("name", placeholder=_("Duomenų rinkinio kodinis pavadinimas")),
+            Field("description", placeholder=_("Detalus duomenų rinkinio aprašas")),
+            Field("identifier", placeholder=_("Duomenų rinkinio identifikatorius")),
```

Comment:
Are you sure this is `Duomenų rinkinys` ? Not something else, like:
- The broader `Duomenų išteklius`
- Something else?